src/features/customerView/CustomerViewContainer.tsx

```diff
@@ -144,6 +147,20 @@ const CustomerViewContainer = () => {
         />
       </Modal>
 
+      <Modal isOpen={!!openResendInvoice} toggleModal={() => setOpenResendInvoice(undefined)}>
+        <SendInvoiceFormContainer
+          orderId={openResendInvoice?.orderId ?? ''}
+          email={'aleksi@identio.fi'}
+          isResend={true}
+          refetchQueries={[]}
```

Comment:
Don't we need to refetch the current page's query?

Reply:
It does this via `refetch()` in `onSubmit()`. Not sure why `refetchQueries` wasn't used here, I'll try.

Reply:
Turns out refetchQueries throws an error for all of the usages on this page. It was caused by a bunch of missing `id`'s in `INDIVIDUAL_CUSTOMER_QUERY`.

After fixing that, the refetchQueries still don't work. This could be fixed by upgrading our Apollo dependencies, but I'll make all `onSubmit`s in the view use the `refetch` function instead of using `refetchQueries` for now.

Reply:
All right, the new Apollo release has some major breaking changes, so we can upgrade it later in a separate task.